operator/pkg/translate/translate_common.go

```diff
@@ -24,6 +25,36 @@ import (
 	binversion "istio.io/istio/operator/version"
 )
 
+const (
+	// Legacy addon components
+	// TODO: move this to versioned config file
```

Comment:
suggestion: create a version specific names-1.5.yaml which defines all components and whether they are legacy or addon.